src/sentry/incidents/models.py

```diff
@@ -373,6 +375,10 @@ class AlertRule(Model):
     threshold_type = models.SmallIntegerField(null=True)
     resolve_threshold = models.FloatField(null=True)
     threshold_period = models.IntegerField()
+
+    user = FlexibleForeignKey(settings.AUTH_USER_MODEL, null=True)
+    team = FlexibleForeignKey("sentry.Team", null=True)
```

Comment:
If the user or team is deleted, should this rule be deleted? Or do we just want to set the column to `null`?

Reply:
The rule is kept with these columns set to `null`, as per the current spec the user will just get a warning when deleting a team (or removing a team from a project)